test/ComposerRequireCheckerTest/FileLocator/LocateComposerPackageSourceFilesTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace ComposerRequireCheckerTest\FileLocator;
+
+use ComposerRequireChecker\FileLocator\LocateComposerPackageSourceFiles;
+use org\bovigo\vfs\vfsStream;
+use org\bovigo\vfs\vfsStreamDirectory;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @todo We're missing tests for the "classmap" and "files" keys.
```

Comment:
I suggest adding an issue about those. This is possibly something interesting for https://github.com/Roave/BetterReflection to solve.